sci-libs/pytorch/pytorch-1.10.0.ebuild

```diff
@@ -0,0 +1,328 @@
+		#Allow escaping sandbox
+		addread /dev/kfd
+		addread /dev/dri
+		addwrite /dev/kfd
+		addwrite /dev/dri
+
+		ebegin "HIPifying cuda sources"
+		tools/amd_build/build_amd.py
+		eend $?
+
+		export PYTORCH_ROCM_ARCH=$(rocminfo | egrep -o "gfx[0-9]+" | uniq | awk -vORS=';' "{print $1}" | sed 's/;$/\n/') || die
+		sed -e "/set(roctracer_INCLUDE_DIRS/s,\${ROCTRACER_PATH}/include,${EPREFIX}/usr/include/roctracer," \
+			-i cmake/public/LoadHIP.cmake || die
+	fi
+}
+
+src_configure() {
+	local mycmakeargs=(
+		-DWERROR=OFF
+		-DTORCH_BUILD_VERSION=${PV}
+		-DTORCH_INSTALL_LIB_DIR=$(get_libdir)
+		-DBUILD_BINARY=$(usex tools ON OFF)
+		-DBUILD_CUSTOM_PROTOBUF=OFF
+		-DBUILD_PYTHON=$(usex python ON OFF)
+		-DBUILD_SHARED_LIBS=$(usex static OFF ON)
+		-DBUILD_TEST=$(usex test ON OFF)
+		-DUSE_ASAN=$(usex asan ON OFF)
+		-DUSE_CUDA=$(usex cuda ON OFF)
+		-DUSE_NCCL=$(usex cuda ON OFF)
+		-DUSE_SYSTEM_NCCL=OFF
+		-DUSE_ROCM=$(usex rocm ON OFF)
+		-DUSE_FBGEMM=$(usex fbgemm ON OFF)
+		-DUSE_FFMPEG=$(usex ffmpeg ON OFF)
+		-DUSE_GFLAGS=$(usex gflags ON OFF)
+		-DUSE_GLOG=$(usex glog ON OFF)
+		-DUSE_LEVELDB=$(usex leveldb ON OFF)
+		-DUSE_LITE_PROTO=OFF
+		-DUSE_LMDB=$(usex lmdb ON OFF)
+		-DUSE_MKLDNN=$(usex mkldnn ON OFF)
+		-DUSE_MKLDNN_CBLAS=OFF
+		-DUSE_NNPACK=$(usex nnpack ON OFF)
+		-DUSE_NUMPY=$(usex python ON OFF)
+		-DUSE_NUMA=$(usex numa ON OFF)
+		-DUSE_OBSERVERS=$(usex observers ON OFF)
+		-DUSE_OPENCL=$(usex opencl ON OFF)
+		-DUSE_OPENCV=$(usex opencv ON OFF)
+		-DUSE_OPENMP=$(usex openmp ON OFF)
+		-DUSE_TBB=OFF
+		-DUSE_PROF=OFF
+		-DUSE_QNNPACK=$(usex qnnpack ON OFF)
+		-DUSE_REDIS=$(usex redis ON OFF)
+		-DUSE_ROCKSDB=OFF
+		-DUSE_ZMQ=$(usex zeromq ON OFF)
+		-DUSE_MPI=$(usex mpi ON OFF)
+		-DUSE_GLOO=$(usex gloo ON OFF)
+		-DUSE_SYSTEM_EIGEN_INSTALL=ON
+		-DUSE_SYSTEM_PYBIND11=ON
+		-DBUILD_NAMEDTENSOR=$(usex namedtensor ON OFF)
+		-DBLAS=$(usex blas Generic Eigen)
+		-DTP_BUILD_LIBUV=OFF
+		-Wno-dev
+	)
+
+	cmake_src_configure
+
+	if use python; then
+		CMAKE_BUILD_DIR="${BUILD_DIR}" distutils-r1_src_configure
+	fi
+
+	# do not rerun cmake and the build process in src_install
+	sed '/RERUN/,+1d' -i "${BUILD_DIR}"/build.ninja || die
+}
+
+src_compile() {
+	cmake_src_compile
+
+	if use python; then
+		USE_SYSTEM_LIBS=ON CMAKE_BUILD_DIR=${BUILD_DIR} distutils-r1_src_compile
+	fi
+}
+
+src_install() {
+	cmake_src_install
+
+	local LIB=$(get_libdir)
+	if [[ ${LIB} != lib ]]; then
+		mv -fv "${ED}"/usr/lib/*.so "${ED}"/usr/${LIB}/ || die
+	fi
+
+	rm -rfv "${ED}/torch"
+	rm -rfv "${ED}/var"
+	rm -rfv "${ED}/usr/lib"
+
+	rm -fv "${ED}/usr/include/*.{h,hpp}"
+	rm -rfv "${ED}/usr/include/asmjit"
+	rm -rfv "${ED}/usr/include/c10d"
+	rm -rfv "${ED}/usr/include/fbgemm"
+	rm -rfv "${ED}/usr/include/fp16"
+	rm -rfv "${ED}/usr/include/gloo"
+	rm -rfv "${ED}/usr/include/include"
+	rm -rfv "${ED}/usr/include/var"
+
+	cp -rv "${WORKDIR}/${P}/third_party/pybind11/include/pybind11" "${ED}/usr/include/"
+
+	rm -fv "${ED}/usr/${LIB}/libtbb.so"
```

Comment:
```suggestion
	rm -v "${ED}/usr/${LIB}/libtbb.so" || die
```

Reply:
And in the other lines above too.

If external commands are called we need to call `die` if they fail to stop the ebuild there. Also drop the `-f` argument, as this causes rm to exit successfully even if the file does not exist and therefore interferes with `|| die`